deployments/systemd/hooks-default.yaml

```diff
@@ -16,3 +16,7 @@ hooks:
     - workdir: "/etc/nvidia-mig-manager"
       command: "/bin/bash"
       args: ["-x", "-c", "source hooks.sh; apply-exit"]
+    refresh-cdi-config:
```

Comment:
The keys in the hooks struct defines point at which one can inject hooks. I suppose it would be sufficient to trigger a CDI refresh as an `apply-exit` hook?

Reply:
I thought about it, but an individual hook allows users to opt out. if we call a CDI refresh from the `apply-exit` hook users won't be able to opt out of it.

Reply:
The latest version doesn't address the following:
> would probably do what you want it to do -- although, as you point out, we may want to be able to specify environment variables.

Reply:
Given that https://github.com/NVIDIA/nvidia-container-toolkit/pull/1107 is merged, if I remove the `--output` from this PR that should be enough to allow the user to specify custom configuration via the config file